client/src/pages/Lobby/Lobby.tsx

```diff
@@ -93,6 +125,31 @@ const Lobby: FC<LobbyProps> = (props: LobbyProps) => {
             players={game.players}
             currentPlayerId={currentPlayerId}
           />
+          <div>
+            <h4>Cartes</h4>
```

Comment:
The previous heading is a `h2`. This one should either be a `h2` (same level) or a `h3` (lower level). Never skip heading levels!

And I think it should be a `h2` (even if we're going to redesign this section).